attributes/default.rb

```diff
@@ -1,2 +1,3 @@
 default['scrutinizer-whitelist']['source-url'] = 'https://scrutinizer-ci.com/api/meta'
 default['scrutinizer-whitelist']['priority'] = '05'
+default['scrutinizer-whitelist']['ports'] = [22]
```

Comment:
any chance it can depend on sshd cookbook like this does?
https://github.com/inviqa/chef-inviqa-base/blob/master/attributes/default.rb#L10